requirements.txt

```diff
@@ -1,3 +1,3 @@
-torch==1.7.1
-clip @ git+https://github.com/openai/CLIP.git@cfcffb90e69f37bf2ff1e988237a0fbe41f33c04
-jina==2.0.0rc6
+jina==2.0.0rc9
+tensorflow-cpu==2.5.0
```

Comment:
Why do you use tensorflow-cpu instead of tensorflow?

Reply:
This removes the gpu support of tensorflow so please change this in order to support gpu in this executor

Reply:
Nothing about me, I copied from previous repo